.travis.yml

```diff
@@ -3,10 +3,10 @@ install:
   - wget https://s3.amazonaws.com/rebar3/rebar3 && chmod +x rebar3
 language: erlang
 otp_release:
-  - 20.1
-  - 20.0
-  - 19.3
+  - 22.2
+  - 21.3
```

Comment:
This file should be deleted!

Reply:
yes, have to be deleted once the actions are finalised. I could not make the actions run without a PR